pkg/kubeutils/kubeutils.go

```diff
@@ -603,6 +603,9 @@ func (k *KubeUtils) IsDeploymentReady(ctx context.Context, cli client.Client, ns
 		return false, err
 	}
 	if deploy.Spec.Replicas == nil {
+		if deploy.Status.ReadyReplicas == 1 {
```

Comment:
this assumes the deployment has a single replica.

Reply:
Drom docs

```
// Number of desired pods. This is a pointer to distinguish between explicit
// zero and not specified. Defaults to 1.
```